sdk/formrecognizer/perf-tests/README.md

```diff
@@ -0,0 +1,9 @@
+### Guide
+
+1. Build the ai-form-recognizer perf tests package `rush build -t @azure-tests/perf-ai-form-recognizer`.
+2. Copy the `sample.env` file and name it as `.env`.
+3. Create a cognitive services account and populate the `.env` file with the relevant credentials.
+4. Run the tests as follows
```

Comment:
Perhaps link the rate limits service page?

Reply:
I don't believe the limit is stated on docs.microsoft.com. I'll look into this, but for now I've just added a warning.